apps/web/app/(app)/gardens/[chain]/[garden]/[community]/pool/[poolId]/[proposalId]/page.tsx

```diff
@@ -275,14 +275,14 @@ export default function Proposal({
             Proposal passed and executed successfully
           </div>
         ) : (
-          <div className="mt-10 flex justify-evenly">
+          <>
```

Comment:
Don't think there is a need to wrap the component with a fragment, maybe you can return only the component

Reply:
Good catch!